Example/URBNSwiftCarousel/Convenience.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Convenience.swift
+//  URBNSwiftCarousel
+//
+//  Created by Kevin Taniguchi on 4/26/16.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+extension UIColor {
+    static var colors: [UIColor] {
+        return [UIColor.redColor(), UIColor.orangeColor(), UIColor.purpleColor(), UIColor.cyanColor(), UIColor.blueColor(), UIColor.greenColor(), UIColor.yellowColor()]
+    }
+    
+    static func colorForIndex(index: Int) -> UIColor {
+        return colors[index]
+    }
+}
+
+extension UIImage {
+    static func imageFromView(view: UIView) -> UIImage? {
+        UIGraphicsBeginImageContextWithOptions(view.bounds.size, view.opaque, 0.0);
+        guard let ctx = UIGraphicsGetCurrentContext() else { return nil }
+        view.layer.renderInContext(ctx)
+        let image = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        return image
```

Comment:
i couldn't get that to work, and it looks like that method is having some problems http://stackoverflow.com/questions/23157653/drawviewhierarchyinrectafterscreenupdates-delays-other-animations